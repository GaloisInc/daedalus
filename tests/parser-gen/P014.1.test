P014.ddl
--gen
--interp=inputs/P014.1.inp
