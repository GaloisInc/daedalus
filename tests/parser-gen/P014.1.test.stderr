TODO: TC (TCAnnot {tcAnnot = (SourceRange {sourceFrom = SourcePos {sourceIndex = 52, sourceLine = 6, sourceColumn = 11, sourceFile = "./P014.ddl"}, sourceTo = SourcePos {sourceIndex = 56, sourceLine = 6, sourceColumn = 15, sourceFile = "./P014.ddl"}},Annot {annotStates = [], annotContx = Nothing}), tcAnnotExpr = TCSetAny})
CallStack (from HasCallStack):
  error, called at src/Daedalus/ParserGen/Action.hs:808:10 in daedalus-0.1.0.0-inplace:Daedalus.ParserGen.Action
